packages/domain/lib/src/models/response/response_model.dart

```diff
@@ -0,0 +1,9 @@
+class ResponseModel {
```

Comment:
naming - ResponseModel to ListResponseModel

Reply:
hmm, and if my json will be an object I will need an ObjectResponceModel?

Reply:
no. in all other cases we will return the concrete model from data (like Movie/User entity). but in this case we need to craete a wrapper over movie list, because we need not only data, but also the header